src/lando/main/models/repo.py

```diff
@@ -38,6 +39,48 @@ class RepoError(Exception):
 class Repo(BaseModel):
     """Represents the configuration of a particular repo."""
 
+    # The path to the repository.
+    path: str
+    vcs: AbstractVcs
+
+    @property
+    def REJECTS_PATH(self) -> Path:
+        return self.vcs.REJECTS_PATH
+
+    def push(
+        self, push_path: str, target: Optional[str] = None, force_push: bool = False
+    ):
+        return self.vcs.push(push_path, target, force_push)
+
+    def last_commit_for_path(self, repo_path: str, path: str) -> bytes:
+        return self.last_commit_for_path(repo_path, path)
+
+    @property
+    def push_target(self):
+        """The target branch or bookmark to push to."""
+        return self.push_bookmark or None
```

Comment:
Yes, I had a wonder about this before. I thing I renamed bookmark to target in some places, under the assumption that it would be a target _branch_ in git, but I'm not 100% certain that's correct. Is it? What's a bookmark?

Reply:
A bookmark is the Mercurial equivalent of a Git branch. In the Firefox Mercurial repos we don't use bookmarks, instead the `tip` commit on the `default` branch is always the only head, and we have a hook to enforce a single head per branch. For other Mercurial repos, such as v-c-t, there is an `@` bookmark which is essentially the equivalent of the default/main branch in a Git repo. It's the default bookmark which is checked out when the repo is first cloned.

For the Git landing worker, the equivalent push target would be a Git branch. I think we built this concept into Lando specifically for v-c-t, which is the only Mercurial repo where we wanted to maintain the `@` bookmark.

Reply:
Ok, so the `push_bookmark` is a column in the repo model.

Now that we have Git support, perhaps renaming it to `push_target` would be more SCM-agnostic (e.g., it would map to either an Hg bookmark or a Git branch).

However, is it worth the effort renaming and adding a migration?

@cgsheeh suggested the other that we should strive to get a clean DB to prod on the first prod-prod day of use, rather than having to migrate over time. So maybe now's the right time to migrate?

Reply:
`push_target` sounds like a reasonable vcs-agnostic choice to me. :+1: 

> @cgsheeh suggested the other that we should strive to get a clean DB to prod on the first prod-prod day of use, rather than having to migrate over time. So maybe now's the right time to migrate?

This comment was more that we don't have any data in the production DB yet, so we shouldn't be afraid to make changes as we need to as we will not have to deal with actual data/behaviour changes. We will definitely do migrations over time (in fact easier migrations was one of the motivating factors in migrating to Django), but considering the whole DB is about to be wiped, if we need to change fields, this is a reasonable time to do it.